ql/src/java/org/apache/hadoop/hive/ql/ddl/table/AbstractAlterTableOperation.java

```diff
@@ -41,6 +39,11 @@
 import org.apache.hadoop.hive.ql.metadata.Partition;
 import org.apache.hadoop.hive.ql.metadata.Table;
 import org.apache.hadoop.hive.ql.session.SessionState;
+import org.apache.thrift.TException;
+
+import java.util.ArrayList;
```

Comment:
I think spotless will complain that these imports are separated.

Reply:
It does not :). Anyway, I reordered them.

Reply:
ah, yes because it's not the `hive-iceberg` module :)